core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -430,8 +430,8 @@ package object config {
       .doc("The chunk size in bytes during writing out the bytes of ChunkedByteBuffer.")
       .bytesConf(ByteUnit.BYTE)
       .checkValue(_ <= ByteArrayMethods.MAX_ROUNDED_ARRAY_LENGTH,
-        "The chunk size during writing out the bytes of" +
-        " ChunkedByteBuffer should not larger than Int.MaxValue - 15.")
+        "The chunk size during writing out the bytes of ChunkedByteBuffer should" +
+          s" not larger than ${ByteArrayMethods.MAX_ROUNDED_ARRAY_LENGTH}.")
```

Comment:
not larger than => be at most
or => not be greater than

Reply:
ok, thanks